_posts/2022-06-13-aws-rds-iam-authentication-spring-boot.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+authors: [yolan_vloeberghs]
+title: 'RDS IAM Authentication with Spring Boot - secure password-less database authentication on AWS'
+image: /img/2022-06-13-aws-rds-iam-authentication-spring-boot.jpeg
+tags: [Spring Boot, AWS, Spring, Security]
+category: Cloud
+comments: true
+---
+
+- [Introduction](#introduction)
+- [Database role grant](#database-role-grant)
+- [AWS](#aws)
+- [Spring Boot](#spring-boot)
+- [Conclusion](#conclusion)
+
+## Introduction
+Database authentication in Spring Boot has traditionally always been very simple to implement and configure to your needs.
+Whether it is with a custom data source, or just using the `spring.datasource` properties, it was always pretty straight out-of-the-box to get a working database connection.
+Handling the sensitive data such as database passwords in different environments can be tricky, but nowadays, there are enough solutions to tackle this problem in a secure and discrete way.
+
+However, if you are deploying your application on the AWS cloud, you don't have to struggle with your password anymore or need a fancy tool, because you simply don't need a password.
```

Comment:
```suggestion
However, if you are deploying your application on the AWS cloud, you don't have to struggle passwords anymore or need a fancy tool, because you simply don't need a password.
```